acquia/cli
# Release issues related to caching

**Motivation**
Preventing release problems like https://github.com/acquia/cli/issues/651

The problem turned out to be the API command cache. The primary cache is `var/cache/ApiCommands.cache`, but there's a fallback cache at `/tmp/symfony-cache`. On my build machine, `/tmp/symfony-cache` was populated for some reason and so ACLI never needed to populate `var/cache/ApiCommands.cache` during the build.

**Proposed changes**

- The first commit fixes #651 by adding `acli cc` to the manual build steps
- The second commit greatly simplifies building the kernel. This did not directly contribute to #651, it turned out to be a red herring, but it _could_ cause unexpected behavior if end users set `APP_ENV` or `APP_DEBUG`. We don't need this functionality, so let's nuke it.

**Alternatives considered**
Instead of running `ckc` and `cc` separately, we could combine them into a single proto-command in `bin/acli`.

I also considered removing the filesystem fallback cache `/tmp/symfony-cache`, but this seems risky, I don't know what side effects that might have.

**Testing steps**
1. Follow the [contribution guide](https://github.com/acquia/cli/blob/master/CONTRIBUTING.md#building-and-testing) to build `acli.phar` and ensure it runs even on a clean environment (i.e. a Lando container or something with clear `/tmp`)

**Merge requirements**
